CHANGELOG.md

```diff
@@ -19,6 +19,16 @@ cargo-dist has traditionally parsed the version number of new releases and used
 
 * impl @mistydemeo [feat: allow always marking releases as stable](https://github.com/axodotdev/cargo-dist/pull/1054)
 
+## Vendoring external GitHub actions
+
+This feature allows vendoring (almost) all external GitHub actions called by the release workflow. When enabled, specific known versions of all external actions (except `actions/checkout`) are copied into your repository as a part of `cargo dist init`. We don't expect most users to need this feature, but it's provided for users with specific security needs.
+
+To enable this user, set the [`vendor-workflow-deps`](https://opensource.axo.dev/cargo-dist/book/reference/config.html#vendor-workflow-deps) option in your `Cargo.toml` to `true`.
```

Comment:
i think the word `user` here is errant maybe?

Reply:
Good catch, I'll swap to `setting`.